src/app/qgsguivectorlayertools.h

```diff
@@ -81,3 +81,4 @@ class QgsGuiVectorLayerTools : public QgsVectorLayerTools
 };
 
 #endif // QGSGUIVECTORLAYERTOOLS_H
+
```

Comment:
```suggestion
```

Try to avoid introducing changes in unrelated files.

Reply:
Side effect of reusing an old branch, should be fixed.